src/Controls/src/Core/VisualElement.cs

```diff
@@ -1159,6 +1161,27 @@ static void FlowDirectionChanged(BindableObject bindable, object oldValue, objec
 			(bindable as IPropertyPropagationController)?.PropagatePropertyChanged(VisualElement.FlowDirectionProperty.PropertyName);
 		}
 
+		static object CoerceIsEnabledProperty(BindableObject bindable, object value)
```

Comment:
Something in here also needs to expose a something for derived things - like buttons that have Commands that support CanExecute.

Reply:
Now using `IsEnabledCore`